cabalscan/src/CabalScan/RuleGenerator.hs

```diff
@@ -164,9 +175,10 @@ generateRule
   -> ComponentType
   -> Text
   -> Attributes
+  -> Attributes
```

Comment:
I'm not convinced of adding a generic list of attributes only to hold `main_is`. This could be done more directly with an argument of type `Maybe FilePath`.

Perhaps you are anticipating the need to add other attributes later on. I think we could defer that concern until the time comes.

Reply:
That's what I was addressing under @googleson78 PR. In the end I think there will be no more cases like this one (`main-is`). I'll fix it to `Maybe`.